workflows/wftest.yml

```diff
@@ -0,0 +1,23 @@
+name: Periodic workflow test
```

Comment:
I think this looks good in principle, but we should run another workflow (ideally the test portion of the workflow in the iwc repo) here via [reusable workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#creating-a-reusable-workflow). I don't think we'll want to keep separate copies that we have to update with each new version of the parent workflow. The workflow also lacks all of the caching steps, I think we want to keep this, which should be doable with the reusable workflow setup. That's a bit more work but I think it'll pay off pretty soon.

Reply:
Here's the [reusable workflow](https://github.com/galaxyproject/iwc/blob/f7a3e1e6c49bc1c6afd6a056318d0154c05db715/.github/workflows/test_workflows.yml) for galaxy workflow testing. For now I've tested it with a [caller](https://github.com/simleo/sars-cov-2-se-illumina-wgs-variant-calling/blob/dda19bbb989be432909d6540ae2887c0aa47bc1d/.github/workflows/wftest.yml) written for individual `iwc-workflows` repos, in https://github.com/simleo/sars-cov-2-se-illumina-wgs-variant-calling. It seems to be working fine. I still have to check its integration in the main `iwc` workflow.

Reply:
That looks amazing, thanks so much @simleo!